pathways/shipments.py

```diff
@@ -479,53 +479,78 @@ def add_pest_clusters(config, shipment):
         cluster_sizes = _infested_stems_to_cluster_sizes(
             infested_stems, max_infested_stems_per_cluster
         )
-        for cluster_size in cluster_sizes:
-            # Generate cluster as indices in the array of stems
-            distribution = config["clustered"]["distribution"]
-            if distribution == "gamma":
-                param1, param2 = config["clustered"]["parameters"]
+        cluster_indexes = []
+        distribution = config["clustered"]["distribution"]
+        if distribution == "gamma":
+            param1, param2 = config["clustered"]["parameters"]
+            gamma_min_max = stats.gamma.interval(0.999, param1, scale=param2)
+            max_width = gamma_min_max[1] - gamma_min_max[0]
```

Comment:
Since this is how you are using `gamma_min_max` (`gamma_min_max[1] - gamma_min_max[0]`), you could do

```
gamma_min, gamma_max = stats.gamma.interval(...)
```

More variables, but more readable code.